ui/nuxeo-filter.js

```diff
@@ -269,6 +269,15 @@ import { FiltersBehavior } from './nuxeo-filters-behavior.js';
 
     _forwardHostPropV2(prop, value) {
       if (this._instance) {
+        const pendingUpdate = this._isPropertyPending('document') || this._isPropertyPending('user');
+        if (pendingUpdate) {
+          // if we have a pending update
+          const toClear = !this.check(); // let's check if instance will be removed
```

Comment:
This seems to be the approach to go. What about the idea of caching the `this.check()` result?

Reply:
as all caching... tricky invalidation... check only happens when `document` or `user` are pending so not for all host properties thus not that costly 🤞

Reply:
🤞